hook/vaas/hook.go

```diff
@@ -110,29 +110,13 @@ func (sh *Hook) RegisterBackend(taskInfo mesosutils.TaskInfo) error {
 	}
 
 	if taskInfo.GetLabelValue(vaasAsyncLabelKey) == "true" {
-		taskURI, err := sh.client.AddBackend(backend, true)
-		if err != nil {
-			return fmt.Errorf("could not register with VaaS director: %s", err)
-		}
-		log.Info("Waiting for successful Varnish configuration change...")
-		sh.backendID = backend.ID
-
-		task := &Task{
-			ResourceURI: taskURI,
-			Status:      StatusPending,
-		}
-		err = sh.watchTaskStatus(task)
-		if err != nil {
-			return fmt.Errorf("could not register with VaaS director: %s", err)
-		}
-	} else {
-		_, err := sh.client.AddBackend(backend, false)
-
-		if err != nil {
-			return err
-		}
-		sh.backendID = backend.ID
+		return fmt.Errorf("async VaaS registration is no longer supported")
```

Comment:
Here I would add a warning log and simply register synchronously. It does not make sense (for me) to fail the deployment of the application.

Reply:
If you allow async-label and register it synchronously then nobody will remove this label